app/src/main/java/org/fossasia/openevent/general/settings/SettingsFragment.kt

```diff
@@ -29,13 +32,18 @@ class SettingsFragment : PreferenceFragmentCompat(), PreferenceChangeListener {
         preferenceChange(evt)
     }
 
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+        val view = super.onCreateView(inflater, container, savedInstanceState)
+        view?.setBackgroundColor(Color.WHITE)
```

Comment:
why not in xml?

Reply:
This cant be done in settings.xml file actually

Reply:
why?

Reply:
That xml file contains a PreferenceScreen, not a layout. Preference screen doesnt have any xml method to set the background. We do it programitically :+1: